src/main/java/model/Result.java

```diff
@@ -0,0 +1,37 @@
+package model;
+
+import exception.Message;
+import java.util.List;
+
+public class Result {
+
+    private final List<Prize> prizes;
+
+    private Result(final List<Prize> prizes) {
+        this.prizes = prizes;
+    }
+
+    public static Result of(final List<String> values, final int personCount) {
+        List<Prize> prizes = values.stream()
+                .map(Prize::new)
+                .toList();
+        validate(values, personCount);
+        return new Result(prizes);
+    }
```

Comment:
values는 조금 친절하게 어떤 value인지 작성해줘야 좋겠네요.

```suggestion
    public static Result of(final List<String> prizeInputs, final int personCount) {
        List<Prize> prizes = prizeInputs.stream()
                .map(Prize::new)
                .toList();
        validate(prizeInputs, personCount);
        return new Result(prizes);
    }
```

Reply:
List<Prize>를 먼저 생성하고, 검증해야하는 이유가 있나요?

Reply:
이유는 없습니다! 둘의 순서를 바꾸는 것이 더 괜찮아보이네요.👍